dashboard/test/ui/features/learning_platform/lesson_extras_teacher_panel.feature

```diff
@@ -22,4 +22,4 @@ Feature: Lesson extras teacher panel
     And check that the URL contains "section_id="
     And check that the URL contains "user_id="
     And I wait until element ".uitest-sectionselect:contains(Untitled Section)" is visible
-    And I wait until element "td:eq(1)" contains text "Sally"
\ No newline at end of file
+    And I wait until element "td:eq(1)" contains text "Sally"
```

Comment:
nit: it looks like there isn't a change in this file other than the new line at the end of the file?

Reply:
Ya there isn't I accidentally made a change and when I fixed it back it added the new line. Not sure that's an issue since we usually want the empty line?

Reply:
I can remove it but ya there was not meant to be a change here